README.md

```diff
@@ -0,0 +1,48 @@
+# Analytics
+
+Interface web para consulta de indicadores de acessos e publicação das coleções da rede SciELO.
+
+
+### Arquivos: Dockerfile* e docker-compose*.yml
+
+
+Dockerfile:
+- **Dockerfile**: contém as definições para construir a imagem pronta para instalar em **produção**
+- **Dockerfile-dev**: contém as definições para construir a imagem pronta para instalar em **desenvolvimento**
+
+Docker Compose:
+- **docker-compose.yml**: contém as definições para iniciar todos os containers necessários para rodar em **produção**
+- **docker-compose-dev.yml**: contém as definições para iniciar todos os containers necessários para rodar em **desenvolvimento**
+
+
+### Como executar os tests
+
+
+
+- Para rodar os tests de unidade, pode executar: ``python setup.py test``
```

Comment:
Como seria rodar dentro do container? Seria algo como: `docker exec -it analytics_webapp_1 python setup.py test`. Se sim, fazendo aqui ele acusa 

```
running test
Searching for coverage
Reading https://pypi.python.org/simple/coverage/
Couldn't find index page for 'coverage' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
No local packages or download links found for coverage
error: Could not find suitable distribution for Requirement.parse('coverage')
```

Reply:
@rafaelpezzuto precisa de mais ajustes para rodar os testes, irei encorporar  esses ajuste e em outro momento ajustamos isso.